lib/resources/r4/related_person_patch.yaml

```diff
@@ -64,3 +64,37 @@ operations:
         <li>Must include a time component with a timezone when specifying a `period`.</li>
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
+
+  - name: test-name-id
+    path: /name/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the value of the `HumanName.id` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "path": "/name/0/id",
+        "op": "test",
+        "value": "CI-12724068-12724066-0"
+      }
+    note: >
+      <ul>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+      </ul>
+  - name: replace-name-family
+    path: /name/{index}/family
+    operation: replace
+    type: String
```

Comment:
I believe this needs to be lowercase 'string' to complete the hyperlink.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/c9bce63c67bb7f03d1c33b644945dfaa15d064db#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR87

Reply:
Do you mind posting a screenshot showing the hyperlink works now?

Reply:
<img width="963" alt="Screenshot 2021-01-21 at 5 51 27 PM" src="https://user-images.githubusercontent.com/56925992/105350547-5c706000-5c11-11eb-8ea5-9cb2106308ae.png">
<img width="1592" alt="Screenshot 2021-01-21 at 5 51 41 PM" src="https://user-images.githubusercontent.com/56925992/105350568-609c7d80-5c11-11eb-814a-d638ebe7accf.png">